docs/source/getting-started/quickstart.rst

```diff
@@ -45,7 +45,7 @@ Enable ``contrib`` globally with the following snippet:
 
 .. tab-set::
 
-    .. tab-item:: Python
+    .. tab-item:: Python (Context API)
```

Comment:
Not absolutely blocking this, but we need to think about what this looks like for an absolutely new user: They don't know anything about OpenDP at this point, and the first thing we do is throw two different APIs at them. Are they supposed to look at both?

If the framework API is something they don't even need to worry about, I'm fine replacing it with a Polars example; I'm just really not sure about showing both.